nextflow_schema.json

```diff
@@ -2,67 +2,265 @@
                 },
-                "email": {
+                "phenotype": {
+                    "type": "string",
+                    "description": "Phenotype CSV file specifying the experimental design for DESeq2.",
+                    "fa_icon": "fas fa-file-csv",
+                    "mimetype": "text/csv",
+                    "help_text": "There are two rules for providing the phenotype CSV file. 1) The phenotype 'sample' column must match the sample sheets 'sample' column. 2) The response variable containing the phenotype of primary interest in the experiment must have the column name condition.  All other columns included in the file are controlled for in the `DESeq2` design. \n\n| sample    \t| condition \t| replicate \t|\n|-----------\t|-----------\t|-----------\t|\n| control_1 \t| ctr       \t| 1         \t|\n| control_2 \t| ctr       \t| 2         \t|\n| control_3 \t| ctr       \t| 3         \t|\n| treated_1 \t| trt       \t| 1         \t|\n| treated_2 \t| trt       \t| 2         \t|\n| treated_3 \t| trt       \t| 3         \t|\n\nThe above phenotype file will identify differentially expressed circRNAs/mRNAs between control and treatment cells, whilst controlling for the effect of variation between replicates: ` ~ replicates + condition`",
+                    "pattern": "\\.csv$"
+                }
+            }
+        },
+        "pipeline_options": {
+            "title": "Pipeline Options",
+            "type": "object",
+            "fa_icon": "fas fa-circle-notch",
+            "description": "Main workflow parameters.",
+            "help_text": "Documentation for selecting circRNA quantification tools, analysis modules, outputs etc.",
+            "required": ["tool", "module"],
+            "properties": {
+                "tool": {
+                    "type": "string",
+                    "fa_icon": "fas fa-wrench",
+                    "description": "Comma separated list of circRNA quantification tools to use. Supported tools: ciriquant, circexplorer2, find_circ, circrna_finder, mapsplice, dcc, segemehl",
+                    "help_text": "Select one or a combination of circRNA quantification tools for the pipeline e.g:\n--tool 'circexplorer2, ciriquant, find_circ'\n\nN.B: Selecting more than one circRNA quantification tool will trigger the circRNA filtering parameter --tool_filter",
+                    "default": "circexplorer2",
+                    "pattern": "^((circexplorer2|circrna_finder|ciriquant|dcc|find_circ|mapsplice|segemehl)?,?)*[^,]+$"
+                },
+                "module": {
+                    "type": "string",
+                    "fa_icon": "fas fa-sliders-h",
+                    "description": "Comma separated list of modules to run: 'circrna_discovery', 'mirna_prediction' & 'differential_expression'.",
+                    "mandatory": "circrna_discovery",
+                    "pattern": "^((circrna_discovery|mirna_prediction|differential_expression)?,?)*[^,]+$",
```

Comment:
See my comments above, compared to your `tool` step, this is only a short list and mostly follows a linear progression it seems, so I would rather turn on/off with boolean flags rather than comma separate list